src/test/java/com/revature/services/CurriculumServiceTest.java

```diff
@@ -25,6 +25,7 @@
 	@Mock
 	private CurriculumRepository crMock;
 	private Curriculum curriculum;
+
```

Comment:
Can we take this extra line out? We shouldn't need CurriculumServiceTest in this PR